contracts/customOracle/FlippeningRatioOracleInstance.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.5.16;
+
+import "../interfaces/IOracleInstance.sol";
+import "./FlippeningRatioOracle.sol";
+import "synthetix-2.43.1/contracts/Owned.sol";
+
+contract FlippeningRatioOracleInstance is IOracleInstance, Owned {
+    FlippeningRatioOracle public flippeningRatio;
+    string public targetName;
+    uint256 public targetOutcome;
+
+    bool public outcome;
+    bool public resolvable;
+
+    bool private forcedOutcome;
+    string public eventName;
+
+    constructor(
+        address _owner,
+        address _flippeningRatio,
+        string memory _targetName,
+        uint256 _targetOutcome
+    ) public Owned(_owner) {
+        flippeningRatio = FlippeningRatioOracle(_flippeningRatio);
+        targetName = _targetName;
+        targetOutcome = _targetOutcome;
+    }
+
+    function getOutcome() external view returns (bool) {
+        if (forcedOutcome) {
+            return outcome;
+        } else {
+            return flippeningRatio.getRatio() >= targetOutcome;
+        }
+    }
+
+    function setOutcome(bool _outcome) public onlyOwner {
+        outcome = _outcome;
+        forcedOutcome = true;
+    }
+
+    function clearOutcome() public onlyOwner {
+        forcedOutcome = false;
+    }
+
+    function setFlippeningRatio(address _flippeningRatio) public onlyOwner {
+        flippeningRatio = FlippeningRatioOracle(_flippeningRatio);
+    }
+
+    function setResolvable(bool _resolvable) public onlyOwner {
```

Comment:
this can always return true as it uses CL feeds and not jobs so no sanity check needed